test/Makefile.am

```diff
@@ -24,10 +24,10 @@ unitTest_SOURCES = \
 	unitTest.cpp
 
 # List libraries to link into binary
-unitTest_LDADD = ../src/libCoinUtils.la
+unitTest_LDADD = ../src/libCoinUtils.la $(COINUTILSLIB_LFLAGS)
```

Comment:
It shouldn't be necessary to repeat the flags used to link the CoinUtils lib here.